build.gradle

```diff
@@ -116,13 +118,13 @@ signing {
     useInMemoryPgpKeys(System.getenv("SIGNING_KEY"), System.getenv("SIGNING_PASSWORD"))
     sign publishing.publications.mavenJava
 }
-
-nexusStaging {
-    username = System.getenv("MAVEN_USERNAME")
-    password = System.getenv("MAVEN_PASSWORD")
+nexusPublishing {
+    repositories {
+        sonatype {  //only for users registered in Sonatype after 24 Feb 2021
+            username = ossrhUsername
+            password = ossrhPassword
+            nexusUrl.set(uri("https://s01.oss.sonatype.org/service/local/"))
+            snapshotRepositoryUrl.set(uri("https://s01.oss.sonatype.org/content/repositories/snapshots/"))
+        }
+    }
 }
-
-// Don't publish snapshots to Maven Central
-closeAndReleaseRepository.onlyIf { isReleaseVersion }
```

Comment:
Shouldn't we do similar things for Nexus publishing (to publish releases only)

Reply:
could be possible, just the tasks having different names